src/core/contrib.js

```diff
@@ -25,7 +29,14 @@ function findUserURLs(...thingsWithUsers) {
 
 async function toHTML(urls, editors, element, headers) {
   const args = await Promise.all(
-    urls.map(url => fetch(new Request(url, { headers })).then(r => r.json()))
+    urls
+      .map(url =>
+        fetch(new Request(url, { headers })).then(r => {
+          if (checkLimitReached(r)) return;
```

Comment:
Worried that returning undefined here will break things. Make sure you check and return `null` or `{}` so whatever is expecting a result doesn't crash.